src/App.tsx

```diff
@@ -1,4 +1,12 @@
-function App() {
-  return <div></div>;
+import AppContent from "./components/appcontent";
+
+function App ({ appContent }: { appContent: any }) {
```

Comment:
instead of comparing proptypes like this, do it the way you have done in appContext.tsx